.github/workflows/pr-workflow.yml

```diff
@@ -6,20 +6,10 @@ jobs:
     name: Gradle Build
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v2
         with:
-          java-version: 11
-      - uses: actions/cache@v1
```

Comment:
FWIW, [`cache` is not recommended when using the `gradle-build-action`](https://github.com/gradle/gradle-build-action#why-use-the-gradle-build-action):
> When using gradle-build-action we recommend that you not use actions/cache or actions/setup-java@v2 to explicitly cache the Gradle User Home. Doing so may interfere with the caching provided by this action.